modules/ELT/pages/getting-started-with-airbyte.adoc

```diff
@@ -0,0 +1,197 @@
+= Airbyte with Teradata Vantage
+:experimental:
+:page-author: Krutik Pathak
+:page-email: krutik.pathak@teradata.com
+:page-revdate: June 1st, 2023
+:description: Use Airbyte with Teradata Vantage.
+:keywords: airbyte, data warehouses, compute storage separation, teradata, vantage, cloud data platform, object storage, data transfer, data extraction, enterprise analytics, elt.
+:tabs:
+:dir: getting-started-with-airbyte
+
+== Overview
+
+This tutorial showcases how to use Airbyte (an Open-source Extract Load Transform Tool) with Teradata Vantage. We work with a very simple end-to-end setup to load the employees’ data from Google Sheets to Teradata Vantage using Airbyte.
+
+
+image::{dir}/sample_employees_payrate_google_sheets.png[Sample Employees Payrate Google Sheets, width=75%]
+
+* Source: Google Sheets
+* Destination: Teradata Vantage
+
+== Prerequsites
+* Access to Teradata Vantage Instance. As this will be our destination. You will need a database host, username, and password for Airbyte’s configuration.
++
+include::ROOT:partial$vantage_clearscape_analytics.adoc[]
+
+* docker-compose to run the link:https://github.com/airbytehq/airbyte[Airbyte open-source] locally. docker-compose comes with Docker Desktop. Please refer to link:https://docs.docker.com/compose/install/[docker docs] for more details. 
+
+* link:https://support.google.com/googleapi/answer/6158841?hl=en[Google Cloud Platform API enabled for your personal or organizational account]. You’ll need to authenticate your Google account via OAuth or via Service Account Key Authenticator.
+
+* Data from the source system. Here, in this case, we take a link:https://docs.google.com/spreadsheets/d/1XNBYUw3p7xG6ptfwjChqZ-dNXbTuVwPi7ToQfYKgJIE/edit#gid=0[sample employee pay rate from google sheets].
+
+== Launch Airbyte Opensource
+* Clone the Airbyte OpenSource repository and go to the airbyte directory.
++
+[source, bash]
+----
+git clone --depth 1 https://github.com/airbytehq/airbyte.git
+cd airbyte
+----
+
+Make Sure to have Docker Desktop running before running the shell script `run-ab-platform`.
+
+* Run the shell script `run-ab-platform` as
+
++
+[source, bash]
+----
+./run-ab-platform.sh
+----
+
+* Log in to the web app http://localhost:8000/ by entering the default credentials found in the `.env` file included in the repository.
+
++
+[source, bash]
+----
+BASIC_AUTH_USERNAME=airbyte
+BASIC_AUTH_PASSWORD=password
+----
+
+After you login with the above credentials, it will ask you for your email and specify your preferences for product improvements.
+
+image::{dir}/specify_preferences.png[Specify Preferences, width=75%]
+
+For more details, please visit link:https://github.com/airbytehq/airbyte[Airbyte github]. Once the Airbyte OpenSource is launched you will see a connections dashboard. If you launched Airbyte open source for the first time, it would not show any connections.
```

Comment:
Maybe _For more details, please visit link:https://github.com/airbytehq/airbyte[Airbyte github]_ should be an info callout?

Reply:
Airbyte has the documentation to launch the Airbyte locally in Windows. I have the link to that page as a note callout.